### ISSUE TITLE
SQL Server CE doesn't recognize CONCAT

### ISSUE THREAD

**Issue by Nathan0710** [FransBouma/Massive#319]
The function is not recognized by SQL Server Compact. [ Name of function = CONCAT,Data type (if known) = ].

**Comment by FransBouma** [comment]
please give a code example in which this failed.

**Comment by diotteo** [comment]
Just passing through, here are examples of SQL CE string concatenation:

SELECT 'FOO' + 'BAR'; --Works
SELECT CONCAT('FOO', 'BAR'); --Fails

SELECT 'FOO' + CAST(1 as NVARCHAR) + 'BAR'; --Works
SELECT CONCAT('FOO', 1, 'BAR'); --Fails

**Comment by FransBouma** [comment]
Yeah I get that :) But I was after an example which uses massive with a string concat on CE that fails. Executing 'Concat' directly on CE might not work, but where is it originating from?

**Comment by diotteo** [comment]
Ah ok. I just found this issue while searching for a way to concat strings on SQL CE, saw the issue had been opened for years and figured I’d provide an example (along with a working fix :) ) in case that was helpful.

Have a good day!

**Comment by FransBouma** [comment]
@diotteo :) It's ok, to be honest I haven't looked at the repo for a long time. This issue is already years old :) It's likely some function which emits the CONCAT string, but haven't looked. In any case, thanks for the example :)
